### DISCUSSION TITLE
Default Password after installation not working?

### DISCUSSION THREAD

**Discussion by jpasley** [Resgrid/Core#111] [General]
Sorry guys for asking this, but 2 things I could not figure out using the documentation.

1. How to access the default resgrid.config settings to verify that the settings are accurate?
2. The default admin login password "changeme1234" is not working after a new docker install. I only have the one Ubuntu server running and I can get to everything else, but setting up the firs time login

**Comment by ucswift** [comment]
Hey, @jpasley, Can you run through this setup https://docs.resgrid.com/get-started/quick-start and see if it helps you? If you used the other documentation, it is a bit out of date. 

For the Docker install, all the configuration variables are done via Environment variables. You can see them here https://github.com/Resgrid/resgrid-setup/blob/master/resgrid.env basically you'll modify that resgrid.env file to what you need.

**Reply by jpasley** [reply]
The new documentation helped a lot! Is there a way to change the web ports? I am already using 80 and 443 on another port forwarded system

**Reply by jpasley** [reply]
I am using the new instructions on a new Ubuntu and Docker build. All ports are forwarded and open and getting an invalid Host Header error on install?

**Reply by jpasley** [reply]
![Screenshot 2023-12-17 160401](https://github.com/Resgrid/Core/assets/9251562/16e18c0f-31dc-480d-ab44-1249cf848d4f)

**Reply by ucswift** [reply]
Hey @jpasley which container is emitting that error?

**Comment by jpasley** [comment]
Its not a container that I can see. Everything works great until I get to the Lets Encrypt section of the instructions and then I get a "bad host header" message and the installation fails.

**Comment by jpasley** [comment]
When I installed Portainer. I see that buildx_buildit_default container is in exited status

**Comment by ucswift** [comment]
You can spin up each container individually; you just reuse the same env file for each. I'm going to fire up and test it out to see if I can repo.

**Comment by ucswift** [comment]
Hey @jpasley, sorry for the delay on this. It took a long time to catch up from the holidays. I just ran through a fresh setup in my lab env and cannot repo this issue. 

Ubuntu 22.04 LTS, Docker-CE, Docker-Compose.

I do have a note to document the container by container setup vs docker-compose as well. But it's gonna take me some time to circle back on that.

**Comment by jpasley** [comment]
Mr. Jackson,

Thank you for the feedback. We are an IT Managed Services provider and we provide assistance at times with Public Safety entities. I saw this application and from my time as a firefighter, EMT, and Police Officer, I thought it had value in being able to at least host it for potential customers or existing customers. We wanted to stage it in our own environment to test it. I know there is a free version, but we wanted to see all the features. Hopefully in the future it can be a seamless setup process that we can implement.

Also, If we needed it setup on an environment, what is the cost to request that?

I look forward to hearing back on this,


Jacob W Pasley - CEO / Founder

Direct: 1-850-600-0860

Email: ***@***.******@***.***>



Schedule a Meeting with Jacob<https://venterprise.net/meetings/jpasley/jacob-pasley-meeting-link>
________________________________
